ncm-dirperm/src/main/pan/components/dirperm/schema.pan

```diff
@@ -30,11 +30,13 @@ function dirperm_permissions_valid = {
 };
 
 type structure_dirperm_entry = {
-    'path'    : string with match(SELF, '^/')
-    'perm'    : string with match(SELF,'[0-7]{3,4}')
-    'owner'   : string with match(SELF, '\w+(:\w+)?')
-    'type'    : string with match(SELF, 'd|f')
-    'initdir' ? string[]
+    'path'       : string with match(SELF, '^/')
+    'perm'       : string with match(SELF,'[0-7]{3,4}')
+    'owner'      : string with match(SELF, '\w+(:\w+)?')
+    'type'       : string with match(SELF, 'd|f')
+    'initdir'    ? string[]
+    'checkmount' : boolean = false
+    'notroot'    : boolean = false
```

Comment:
I'm not sure we can, because there is no default that won't be breaking for others. It would mean "check this directory is on the root filesystem". If we set the default to true, it would break things for people that were using dirperm to create directories within mounts, and if we set it to false then it's going to break for everything on the root fs.